src/RokuAdapter.ts

```diff
@@ -639,9 +639,29 @@ export class RokuAdapter {
         });
     }
 
-    private expressionRegex = /([\s|\S]+?)(?:\r|\r\n)+brightscript debugger>/i;
-    private isObjectCheckRegex = /<.*:\s*(\w+\s*\:*\s*\w*)>/gi;
-    private getFirstWordRegex = /^([\w.\-=]*)\s/;
+    /**
+     *
+     * @param value
+     */
+    private getExpressionDetails(value: string) {
+        return new RegExp(/([\s|\S]+?)(?:\r|\r\n)+brightscript debugger>/i).exec(value);
+    }
+
+    /**
+     * Runs a regex to check if the target is an object and get the type if it is
+     * @param value
+     */
+    private getHighLevelTypeDetails(value: string) {
+        return new RegExp(/<.*:\s*(\w+\s*\:*\s*[\w\.]*)>/gi).exec(value);
+    }
+
+    /**
+     * Runs a regex to get the first work of a line
+     * @param value
+     */
+    private getFirstWord(value: string) {
+        return new RegExp(/^([\w.\-=]*)\s/).exec(value);
+    }
 
```

Comment:
Changed the above into functions to make it a bit harder to accidentally break a regex that is used in many places by simply adding a g flag.